meta/etcd.go

```diff
@@ -0,0 +1,100 @@
+// MIT License. Copyright (c) 2020 CQFN
+// https://github.com/cqfn/degitx/blob/master/LICENSE
+
+package meta
+
+import (
+	"context"
+	"time"
+
+	"cqfn.org/degitx/discovery"
+	"cqfn.org/degitx/misc"
+
+	ma "github.com/multiformats/go-multiaddr"
+	"go.etcd.io/etcd/clientv3"
+)
+
+type etcd struct {
+	Endpoints []string
+}
+
+func NewEtcd(endpoints ...string) (Storage, error) {
```

Comment:
@stepanvalyavskiy please add comments to public functions (maybe we can enforce it in linters?)

Reply:
added, it's also forced with golangci-lint